### PR TITLE
Improve textView on colorlcd

### PR DESCRIPTION
**Pull request by 3djc**
Before
image

With those changes
image

### CODE REVIEW

**Comment by lshems** [comment]
Still a doubling on the overflow?

It at end of one line and start if next?

**Comment by 3djc** [comment]
Or not ... https://github.com/opentx/opentx/blob/2.3/radio/util/lua/lua_event_test.lua

File: radio/src/gui/480x272/bitmapbuffer.h [hunk 1]
```diff
@@ -275,6 +283,16 @@ class BitmapBuffer: public BitmapBufferBase<uint16_t>
 
     uint8_t drawCharWithCache(coord_t x, coord_t y, const BitmapBuffer * font, const uint16_t * spec, int index, LcdFlags flags);
 
+    void drawMaxLenText(coord_t x, coord_t y, const char * s, LcdFlags flags, coord_t maxLen)
```
Comment (bsongis) [inline radio/src/gui/480x272/bitmapbuffer.h]: drawTextMaxWidth + maxWidth

File: radio/src/gui/480x272/bitmapbuffer.h [hunk 2]
```diff
@@ -275,6 +283,16 @@ class BitmapBuffer: public BitmapBufferBase<uint16_t>
...
+    {
+      for (int col = LCD_COLS; 0 < col; col--) {
+        if (getTextWidth(s, col, flags) < maxLen) {
```
Comment (bsongis) [inline radio/src/gui/480x272/bitmapbuffer.h]: <=

File: radio/src/gui/480x272/bitmapbuffer.h [hunk 3]
```diff
@@ -275,6 +283,16 @@ class BitmapBuffer: public BitmapBufferBase<uint16_t>
 
     uint8_t drawCharWithCache(coord_t x, coord_t y, const BitmapBuffer * font, const uint16_t * spec, int index, LcdFlags flags);
 
+    void drawMaxLenText(coord_t x, coord_t y, const char * s, LcdFlags flags, coord_t maxLen)
+    {
+      for (int col = LCD_COLS; 0 < col; col--) {
```
Comment (bsongis) [inline radio/src/gui/480x272/bitmapbuffer.h]: col > 0

**Pull request merged**